docs/setup-cluster/mldes/_index.rst

```diff
@@ -0,0 +1,27 @@
+.. _topic_guide_mldes:
+
+############################################################################################
+ Deploy using HPE MLDES (Machine Learning Development Environment Software) Managed Service
+############################################################################################
+
+.. note::
+
+   Determined is referred to as "MLDE" in MLDES documentation and UI.
+
+This section describes how to deploy a Determined cluster using MLDES. MLDES is a fully managed
+service that abstracts the complexity of managing the underlying infrastructure and provides a
+simple and easy-to-use interface to deploy, manage, and work with Determined clusters.
+
+MLDES uses either an AWS or GCP account as a backend for Determined cluster resources. All you need
+to create a new Determined cluster via MLDES is an AWS account with a few pre-requisite IAM roles,
+or a GCP project ID. View the `MLDES Documentation <https://mldes.ext.hpe.com/docs/index.html>`_ for
+more information. A simple guide to creating an MLDE cluster is described as follows:
+
+#. Go to the `MLDES Console <https://mldes.ext.hpe.com>`_ and sign in with your HPE Passport
+   account.
+#. Create a new organization with a unique name and the necessary details.
+#. Create a new Determined cluster by clicking on the "New MLDE Cluster" button.
+#. Once the cluster is provisioned, open the Determined UI by clicking "Open" in the cluster details
+   box.
+#. Use the Determined UI to create and manage experiments, or open a Jupyter notebook to run your
+   own code by clicking on the "Launch JupyterLab" button.
```

Comment:
To meet the style guide and standards, replace your content with the following.

`.. _topic-guide-mldes:

############################################################################################
 Deploy Using HPE Machine Learning Development Environment Software (MLDES) Managed Service
############################################################################################

.. attention::

   MLDES is only available on Determined Enterprise Edition.


**********
 Overview
**********

You can deploy a Determined cluster using `MLDES <https://mlde-demo.det-cloud.net/docs/index.html>`__. 
MLDES is a managed service that simplifies the infrastructure management complexities. 
It offers an intuitive interface for deploying, managing, and utilizing Determined clusters.

*********************
 Prerequisites
*********************

MLDES supports both AWS and GCP as backends for Determined cluster resources. To start,
ensure you have an AWS account with the required IAM roles or a GCP project ID. 
For more information, visit the `documentation <https://mlde-demo.det-cloud.net/docs/index.html>`__.

******************
 Deployment Guide
******************

Access the Console
===================

#. Visit the `MLDES console <https://mldes.ext.hpe.com>`_.
#. Sign in using your HPE Passport account.

Create an Organization
=======================

#. Select **Organization** to open the Organizations panel and create a new organization.
#. Enter a unique name and select **Save**.

Deploy the Cluster
===================

#. Select **Clusters** to open the Clusters panel.
#. Select **New MLDE Cluster** to start creating a new cluster.
#. Enter a unique name.
#. Review the default cluster and resource pool configuration and make any necessary changes.
#. Select **Create Cluster**.

MLDE provisions the cluster and lets you know when it is ready.

Create and Manage Experiments
==============================

#. Once the cluster is ready, select **Open**.
#. To open a Jupyter notebook, **Launch JupyterLab**.

`